src/Storage/ABS/SourceFile.php

```diff
@@ -23,19 +25,24 @@ class SourceFile extends BaseFile implements SourceInterface
     /** @var string[] */
     private $columnsNames;
 
+    /** @var string */
+    private $type;
+
     public function __construct(
         string $container,
         string $filePath,
         string $sasToken,
         string $accountName,
         CsvOptions $csvOptions,
         bool $isSliced,
-        array $columnsNames = []
+        array $columnsNames = [],
+        string $type = self::TYPE_FILE
```

Comment:
Jen otázka. Nebylo by lepší to udělat na dvě classy? SoucerFile a SourceDirectory?

Reply:
Pecka to mě nenapadlo. https://github.com/keboola/php-db-import-export/pull/36/commits/5ba3e2179128f21f44de2993d44b2de96f0f2b98

Reply:
kouknu na to